src/qibocal/cli/autocalibration.py

```diff
@@ -0,0 +1,73 @@
+import datetime
+import json
+from dataclasses import asdict
+
+import yaml
+from qibolab.serialize import dump_runcard
+
+from ..auto.execute import Executor
+from ..auto.history import add_timings_to_meta
+from ..auto.mode import ExecutionMode
+from ..auto.runcard import Runcard
+from .utils import (
+    META,
+    PLATFORM,
+    RUNCARD,
+    UPDATED_PLATFORM,
+    create_qubits_dict,
+    generate_meta,
+    generate_output_folder,
+)
+
+
+def run_autocalibration(card, folder, force, update):
```

Comment:
Just a small note on names: try to be concise, if possible.

In this case, I would call it:
```suggestion
def autocalibrate(card, folder, force, update):
```

It contains the same level of information, and it doesn't sound bad, since it's still an action :)
(in general, identifiers are symbols, so we could even stretch a bit the English vocabulary and force to be verbs things that are not, analogously with names - for as long as it is clear enough)